website/examples/e-commerce/widgets/ResultsNumberMobile.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { connectStats } from 'react-instantsearch-dom';
+
+const ResultsNumberMobile = ({ nbHits }) => (
+  <div>
+    <strong>{nbHits}</strong> results
```

Comment:
It could be nice to better handle no results state (this is not specific to mobile)
![image](https://user-images.githubusercontent.com/2982512/59918750-f1075080-9425-11e9-995b-4e2f91734976.png)

We have resources advsing the user to handle this state
https://www.algolia.com/doc/guides/building-search-ui/going-further/conditional-display/react/?language=react#handling-the-empty-query

Reply:
Yep! Got that one dully noted on my end, design coming soon! :)

Reply:
Handled in #2601.